ref/Boolean_Operators.md

```diff
@@ -181,7 +181,14 @@ This operator is similar to **&gt;** but tests for **less than or equal to**.
 
 ------
 
-#####  Fuzzy comparisons: `~=`, `~!=`, `~<`, `~>`, `~>=`, `~<=`
+####  Fuzzy comparisons:
+
+##### Approximately equal: `‹expr1› ~= ‹expr2›`
+##### Not approximately equal: `‹expr1› ~!= ‹expr2›`
```

Comment:
For me, "a not approximately equal b" is `not(a ~= b)`, and "a approximately not equal b" differs from that a bit, that is at +-epsilon. But from the diagram it is not visible what happens there, and it must not be important. So, I prefer "approximately not equal" as it matches the symbol order `~`, `!`, `= ` in `~!=`.